lib/models/device.coffee

```diff
@@ -183,6 +183,58 @@ exports.get = (name, callback) ->
 	.nodeify(callback)
 
 ###*
+# @summary Get a single device by UUID
+# @public
+# @function
+#
+# @param {String} uuid - device UUID
+# @param {module:resin.models.device~getCallback} callback - callback
```

Comment:
I think you forgot to annotate the callback. This is still referencing `getCallback` and there is no callback annotation above.